### DISCUSSION TITLE
Bringing a cat into a pod

### DISCUSSION THREAD

**Discussion by bluehat** [microCOVID/microCOVID#1020] [Q&A]
I'm in the process of retrieving a cat from a high-risk household. So, I'm trying to figure out what sort of exposure risk that looks like to my pod (and to evaluate various options to mitigate it.)

Here are some of the sources I've found so far:
- Cats [get COVID relatively easily](https://academic.oup.com/emph/article/2020/1/109/5866981) compared to other animals, they don't have many mutations on their ACE2 receptors.
- Cats have [TMPRSS2 enzyme](https://www.sciencedirect.com/science/article/pii/S0042682219302557?via%3Dihub) which [probably enables fast attack](https://www.nature.com/articles/d41586-021-02039-y)
- Cats [can transmit COVID to other cats](https://www.nejm.org/doi/pdf/10.1056/NEJMc2013400). There have been experimental design problems in figuring out if cats can infect humans.
- Cats are [frequently asymptomatic](https://www.oie.int/app/uploads/2021/03/en-factsheet-sars-cov-2.pdf) 
- [According to one study, about 20% of cats in COVID-positive households got COVID, and according to another 67% of them got it if the cat slept in the bed](https://www.bbc.com/news/health-57666245). I am sadly having a tremendously difficult time finding either study, so it is hard to figure out exactly which variant they are discussing. I've written to the head researchers on both papers, and we'll see if either responds.

Does anybody have any ideas how to model this risk? Has anybody seen anything else useful?

**Comment by blanchardjeremy** [comment]
Wow. I'm impressed by the research you've already done! Well done.

If someone on our team has access to any more research than you do, that's amazing. Although I doubt we do.

Have you considered an approach that's based in mostly being in separate rooms for a week or two, rather than trying to quantify the risk?

I know some cats tend to want to do that anyway when moving into a new space. 

Perhaps that's not the most helpful response, but I thought I'd offer it as it seems more reliable than any amount of research (given that you're only encountering this issue one time).

**Reply by bluehat** [reply]
He's clingy and going to hate not being permitted to do whatever he wants, and yeah that is how it goes sometimes. Thought it might be worth asking just in case.

Also an update: the researcher from the paper about cats sleeping in their owner's beds says the paper isn't done yet, and won't be out for another month or two. So it looks like the BBC jumped the gun a bit on that one. The other researcher hasn't gotten back to met yet.

**Comment by bluehat** [comment]
Tab cleanup, here is a list of other research I was working through. I know the chances are long that anybody will ever need this, but, I'd like to save that future hypothetical person some time if possible. **Disclosure: I have not finished reading all of these papers.**

### Case studies and surveys of exposed cats
[A veterinary campus in France.](https://www.sciencedirect.com/science/article/pii/S2352771420302652) Despite close contact with many infected people, no animals of any type had antibodies.
[Another survey in France](https://onlinelibrary.wiley.com/doi/full/10.1111/tbed.13659)
[Sampling of cats in Germany](https://www.mdpi.com/2076-393X/8/4/772/htm)
[Survey of pets in infected households in Texas](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7743065/) 47.1% of 17 cats positive, virus cultivated from one cat. _Preprint_
[A cat in spain that had symptoms](https://www.pnas.org/content/pnas/117/40/24790.full.pdf)
[Cats in Wuhan got COVID](https://www.tandfonline.com/doi/full/10.1080/22221751.2020.1817796)
[Cats and dogs in Italy](https://www.nature.com/articles/s41467-020-20097-0).) Antibodies
[University Hospital of Besançon](https://www.sciencedirect.com/science/article/pii/S2352771420302937)
### Experimental evidence
[Cat to cat tramsmission](https://www.tandfonline.com/doi/full/10.1080/22221751.2020.1833687)
[Cats and dogs](https://www.pnas.org/content/pnas/117/42/26382.full.pdf)
### Modeling
[ACE2 receptors](https://www.nature.com/articles/s41598-020-80573-x)
### Summaries of other studies
[Nature](https://www.nature.com/articles/s41385-020-00340-z)
[Overview of a variety of animal studies](https://link.springer.com/article/10.1007/s11259-020-09781-0)
[Another overview](https://www.mdpi.com/2076-0817/9/7/529/htm)
[Another overview](https://www.nature.com/articles/s41586-020-2787-6#Sec9)
[Another overview](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7323145/)

**Reply by blanchardjeremy** [reply]
Wow! Great work!!

**Comment by bluehat** [comment]
An update: modeling the risk through the cat continues to be an issue because cats have to go to the vet sometimes. Cats can't be masked or vaccinated, so it is very easy for the microcovid count to rack up extremely fast.
